docs/sources/operations/troubleshooting/troubleshooting-ingest.md

```diff
@@ -0,0 +1,1154 @@
+
+* **Check clients** Check your ingestion pipeline for label conflicts.
+
+* **Verify processing** Verify your label processing and transformation rules.
+
+**Properties:**
+
+- Enforced by: Distributor
+- Retryable: No
+- HTTP status: 400 Bad Request
+- Configurable per tenant: No
+
+### Error: `request_body_too_large`
+
+**Error message:**
+
+`Request body too large: <size> bytes, limit: <limit> bytes`
+
+**Cause:**
+
+The HTTP compressed push request body exceeds the configured limit in your gateway/reverse proxy or service that fronts Loki after decompression.
+
+**Default configuration:**
+
+- Loki server max request body size: Configured at the HTTP server level
+- Nginx gateway (Helm): `gateway.nginxConfig.clientMaxBodySize` (default: 4 MB)
+- Alloy default batch size: 1 MB
+
+**Resolution:**
+
+* **Reduce batch size** in your ingestion client:
+
+   ```alloy
+   loki.write "default" {
+     endpoint {
+       url = "http://loki:3100/loki/api/v1/push"
+       
+       batch_size  = 524288  // 512KB in bytes
+       batch_wait  = "1s"
+     }
+   }
+   ```
+
+* **Split large batches** into smaller, more frequent requests.
+
+* **Increase batch limit** Increase the allowed body size on your gateway/reverse proxy.  For example, in the Helm chart set `gateway.nginxConfig.clientMaxBodySize`; default is 4M.
+
+**Properties:**
+
+- Enforced by: Distributor
+- Retryable: No
+- HTTP status: 413 Request Entity Too Large
+- Configurable per tenant: No
+
+## Storage errors
+
+Storage errors occur when Loki cannot write data to its storage backend or the Write-Ahead Log (WAL).
+
+### Error: Disk space full
+
+**Error message:**
+
+`no space left on device`
+
+**Cause:**
+
+Local storage has run out of space. This can affect ingester WAL, chunk cache, or temporary files.
+
+**Default configuration:**
+
+Disk space is not limited by Loki configuration; it depends on your infrastructure provisioning.
+
+**Resolution:**
+
+* **Free up disk space** on Loki instances.
+* **Configure retention policies** to remove old data.
+* **Scale storage capacity** by adding more disk space.
+* **Monitor disk usage** and set up alerts before reaching critical levels.
+* **Review compactor settings** to ensure old data is being cleaned up.
+
+**Properties:**
+
+- Enforced by: Operating system
+- Retryable: Yes (after freeing space)
+- HTTP status: 500 Internal Server Error
+- Configurable per tenant: No
+
+### Error: Chunk write failure
+
+**Error message:**
+
+`failed to store chunks: storage unavailable`
+
+**Common causes and errors:**
+
+* **S3/Object Storage errors:**
+  - `NoSuchBucket`: Storage bucket doesn't exist
+  - `AccessDenied`: Invalid credentials or permissions
+  - `RequestTimeout`: Network or storage latency issues
+
+* **DynamoDB errors:**
+  - `ProvisionedThroughputExceededException`: Write capacity exceeded
+  - `ValidationException`: Invalid data format
+
+* **Cassandra errors:**
+  - `NoConnectionsAvailable`: Cannot connect to Cassandra cluster
```

Comment:
we should remove these, these store types have been deprecated for quite some time now